molecule/additional_flags/upgrade.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Maybe nit-picky but should this be a symlink like it is for `molecule/modify_config`?

Reply:
Please see commit 2e1b063.